datadotworld/__init__.py

```diff
@@ -141,36 +141,39 @@ def query(dataset_key, query, query_type='sql', profile='default',
 def open_remote_file(dataset_key, file_name, profile='default',
                      mode='w'):
     """
-    Open a remote file object that can be used to write to a file in a
-    data.world dataset
+    Open a remote file object that can be used to write to or read from
+    a file in a data.world dataset
 
     Parameters
     ----------
     dataset_key : str
         Dataset identifier, in the form of owner/id
     file_name: str
-        The name of the file to write
-    profile : str, optional
-        Configuration profile (account) to use.
+        The name of the file to open
     mode: str, optional
-        the mode for the file - currently only 'w' (write string) or
-        'wb' (write binary) are supported, any other value will throw
-        an exception
+        the mode for the file - must be 'w', 'wb', 'r', or 'rb' -
+        indicating read/write ('r'/'w') and optionally "binary"
+        handling of the file data.
 
     Examples
     --------
     >>> import datadotworld as dw
     >>>
+    >>> # write a text file
     >>> with dw.open_remote_file('username/test-dataset',
     ...                          'test.txt') as w:
     ...   w.write("this is a test.")
     >>>
+    >>> # write a jsonlines file
     >>> import json
     >>> with dw.open_remote_file('username/test-dataset',
     ...                          'test.jsonl') as w:
     ...   json.dump({'foo':42, 'bar':"A"}, w)
+    ...   w.write("\\n")
     ...   json.dump({'foo':13, 'bar':"B"}, w)
+    ...   w.write("\\n")
```

Comment:
updated JSON lines examples to write the newlines - previously the examples didn't add these characters, resulting in invalid `.jsonl` files